numba/tests/test_np_randomgen.py

```diff
@@ -92,9 +92,13 @@ def check_numpy_parity(self, distribution_func,
                                       test_size, test_dtype)
         numpy_res = distribution_func.py_func(numpy_rng_instance,
                                               test_size, test_dtype)
-
-        np.testing.assert_array_max_ulp(numpy_res, numba_res,
-                                        maxulp=ulp_prec, dtype=test_dtype)
+        # assert_array_max_ulp doesn't compare scalar booleans/ boolean arrays
+        if isinstance(numba_res, bool) or (isinstance(numba_res, np.ndarray)
+                                           and numba_res.dtype == bool):
+            assert np.all(numba_res == numpy_res)
+        else:
+            np.testing.assert_array_max_ulp(numpy_res, numba_res,
```

Comment:
Question: since we are comparing integers here, can we drop the max ULP requirement altogether and use:

https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html

instead?

Reply:
Used `assert_equal` instead since we also have scalars along with arrays.